assets/javascripts/src/components/pages/Contribution.jsx

```diff
@@ -90,7 +90,7 @@ class FormWithRecurring extends React.Component {
                 : 'One-off amount';
 
 
-        return <div className={'contribute-controls contribute-fields contribute-controls--recurring ' + (this.props.amounts.length % 3 ? 'option-button__fours ' : 'option-button__three ')}>
+        return <div className="contribute-controls contribute-fields contribute-controls--recurring option-button__fours">
```

Comment:
I changed both, just for consistency's sake, I don't believe it will have any effect.